python/lsst/daf/butler/tests/_testRepo.py

```diff
@@ -388,32 +349,71 @@ def addDataIdValue(butler: Butler, dimension: str, value: Any, **related: Any):
     except KeyError as e:
         raise ValueError from e
     # Bad keys ignored by registry code
-    extraKeys = related.keys() - (fullDimension.required | fullDimension.implied)
+    extraKeys = related.keys() - fullDimension.graph.dimensions.names
     if extraKeys:
         raise ValueError(
-            f"Unexpected keywords {extraKeys} not found "
-            f"in {fullDimension.required | fullDimension.implied}"
+            f"Unexpected keywords {extraKeys} not found " f"in {fullDimension.graph.dimensions.names}"
```

Comment:
That's black realizing that it can now fit it in one line, but it does not attempt to merge the strings. It's a black oddity that will happen and we need to keep an eye out for.